cg/cli/store.py

```diff
@@ -108,11 +109,19 @@ def _add_new_analysis_to_the_status_api(bundle_obj, status):
 def completed(context):
     """Store all completed analyses."""
     hk_api = context.obj['hk_api']
+    exit_code = 0
```

Comment:
```suggestion
    exit_code = FAIL
```
suggestion something like this to avoid magic numbers